lib/matplotlib/backends/backend_pgf.py

```diff
@@ -722,10 +724,6 @@ class GraphicsContextPgf(GraphicsContextBase):
 ########################################################################
 
 
-def draw_if_interactive():
```

Comment:
This is not a duplicate in the sense that there is a twin definition elsewhere in the same module. However, in the function `pylab_setup` in `lib/matplotlib/backends/__init__.py`, a dummy default function is automatically filled in for `draw_if_interactive` if the module does not define one. I thought this defaulting mechanism was duplicative.

For API consistency, do you think it is better simply _not to define_ functions such as `draw_if_interactive` when they are not applicable? Or do you think dummies should be exported so that the _set of names_ from each backend module is consistent?